packages/edge-login-ui-rn/src/components/modals/TextInputModal.js

```diff
@@ -103,7 +103,7 @@ export class TextInputModal extends React.Component<Props, State> {
       <ThemedModal bridge={bridge} onCancel={this.handleCancel}>
         {title != null ? <TitleText>{title}</TitleText> : null}
         {message != null ? <MessageText>{message}</MessageText> : null}
-        <ThemedTextField
+        <EdgeTextField
```

Comment:
Updating the modal to match our style is a good idea, but it should be its own commit.

Reply:
Return this back with old component